hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ServicesApp.java

```diff
@@ -133,6 +130,8 @@ public interface ServicesApp {
 
     GlobalDynamicProperties globalDynamicProperties();
 
+    GlobalStaticProperties globalStaticProperties();
```

Comment:
Don't we need a provide method for this new class from Dagger to work for @Inject annotation in some module.